### DISCUSSION TITLE
Update from 14.1 to 15 failes

### DISCUSSION THREAD

**Discussion by techsupportkoeln** [kimai/kimai#2892] [On-Premise: Installation & Updates] [answered]
I use kimai on version 1.14.1 with the ExpensesBundle 1.20 and AuditTrailBundle 1.11.1.

I tried to update to version 1.15 and become the following error: 

What i have done:

cd /var/www/kimai2
git fetch --tags
git checkout 1.15
composer install --no-dev --optimize-autoloader

How can i fix this?

`
Installing dependencies from lock file
Verifying lock file contents can be installed on current platform.
Warning from https://flex.symfony.com/versions.json: You are using an outdated version of Flex, please run: composer update symfony/flex
Package operations: 2 installs, 98 updates, 15 removals
  - Downloading composer/package-versions-deprecated (1.11.99.2)
  - Downloading symfony/flex (v1.13.3)
  - Downloading symfony/service-contracts (v2.4.0)
  - Downloading symfony/polyfill-php80 (v1.23.0)
  - Downloading symfony/polyfill-php73 (v1.23.0)
  - Downloading symfony/console (v4.4.25)
  - Downloading doctrine/cache (1.11.3)
  - Downloading doctrine/annotations (1.13.1)
  - Downloading doctrine/persistence (2.2.1)
  - Downloading doctrine/inflector (2.0.3)
  - Downloading doctrine/dbal (2.13.1)
  - Downloading doctrine/common (3.1.2)
  - Downloading doctrine/orm (2.9.2)
  - Downloading symfony/routing (v4.4.25)
  - Downloading symfony/polyfill-ctype (v1.23.0)
  - Downloading symfony/polyfill-intl-normalizer (v1.23.0)
  - Downloading symfony/polyfill-intl-idn (v1.23.0)
  - Downloading symfony/mime (v4.4.25)
  - Downloading symfony/http-foundation (v4.4.25)
  - Downloading symfony/http-client-contracts (v2.4.0)
  - Downloading symfony/event-dispatcher (v4.4.25)
  - Downloading symfony/var-dumper (v4.4.25)
  - Downloading psr/log (1.1.4)
  - Downloading symfony/debug (v4.4.25)
  - Downloading symfony/error-handler (v4.4.25)
  - Downloading symfony/http-kernel (v4.4.25)
  - Downloading symfony/finder (v4.4.25)
  - Downloading symfony/filesystem (v4.4.25)
  - Downloading symfony/dependency-injection (v4.4.25)
  - Downloading symfony/polyfill-php81 (v1.23.0)
  - Downloading symfony/config (v4.4.25)
  - Downloading symfony/var-exporter (v4.4.25)
  - Downloading symfony/cache-contracts (v2.4.0)
  - Downloading symfony/cache (v4.4.25)
  - Downloading symfony/framework-bundle (v4.4.25)
  - Downloading symfony/stopwatch (v4.4.25)
  - Downloading laminas/laminas-zendframework-bridge (1.2.0)
  - Downloading laminas/laminas-eventmanager (3.3.1)
  - Downloading friendsofphp/proxy-manager-lts (v1.0.5)
  - Downloading doctrine/migrations (3.1.2)
  - Downloading symfony/doctrine-bridge (v4.4.25)
  - Downloading symfony/deprecation-contracts (v2.4.0)
  - Downloading doctrine/doctrine-bundle (2.4.2)
  - Downloading doctrine/doctrine-migrations-bundle (3.1.1)
  - Downloading symfony/security-core (v4.4.25)
  - Downloading gedmo/doctrine-extensions (v3.0.5)
  - Downloading phpstan/phpdoc-parser (0.5.4)
  - Downloading jms/serializer (3.12.3)
  - Downloading twig/twig (v3.3.2)
  - Downloading symfony/translation-contracts (v2.4.0)
  - Downloading symfony/twig-bridge (v4.4.25)
  - Downloading symfony/translation (v4.4.25)
  - Downloading symfony/options-resolver (v4.4.25)
  - Downloading symfony/asset (v4.4.25)
  - Downloading kevinpapst/adminlte-bundle (3.5)
  - Downloading league/csv (9.7.1)
  - Downloading league/html-to-markdown (5.0.0)
  - Downloading mpdf/mpdf (v8.0.11)
  - Downloading myclabs/php-enum (1.8.0)
  - Downloading symfony/inflector (v4.4.25)
  - Downloading symfony/property-info (v4.4.25)
  - Downloading nelmio/cors-bundle (2.1.1)
  - Downloading markbaker/matrix (2.1.3)
  - Downloading markbaker/complex (2.0.3)
  - Downloading phpoffice/phpspreadsheet (1.18.0)
  - Downloading laminas/laminas-escaper (2.7.0)
  - Downloading phpoffice/phpword (0.18.2)
  - Downloading sensio/framework-extra-bundle (v6.1.5)
  - Downloading symfony/mailer (v4.4.25)
  - Downloading symfony/amazon-mailer (v4.4.25)
  - Downloading symfony/css-selector (v4.4.25)
  - Downloading symfony/dotenv (v4.4.25)
  - Downloading symfony/expression-language (v4.4.25)
  - Downloading symfony/property-access (v4.4.25)
  - Downloading symfony/polyfill-intl-icu (v1.23.0)
  - Downloading symfony/intl (v4.4.25)
  - Downloading symfony/form (v4.4.25)
  - Downloading symfony/google-mailer (v4.4.25)
  - Downloading symfony/http-client (v4.4.25)
  - Downloading symfony/mailchimp-mailer (v4.4.25)
  - Downloading symfony/mailgun-mailer (v4.4.25)
  - Downloading monolog/monolog (1.26.1)
  - Downloading symfony/monolog-bridge (v4.4.25)
  - Downloading symfony/polyfill-intl-grapheme (v1.23.0)
  - Downloading symfony/postmark-mailer (v4.4.25)
  - Downloading symfony/security-http (v4.4.25)
  - Downloading symfony/security-guard (v4.4.25)
  - Downloading symfony/security-csrf (v4.4.25)
  - Downloading symfony/security-bundle (v4.4.25)
  - Downloading symfony/sendgrid-mailer (v4.4.25)
  - Downloading symfony/serializer (v4.4.25)
  - Downloading symfony/polyfill-uuid (v1.23.0)
  - Downloading symfony/uid (v5.3.0)
  - Downloading symfony/validator (v4.4.25)
  - Downloading symfony/webpack-encore-bundle (v1.11.2)
  - Downloading symfony/yaml (v4.4.25)
  - Downloading symfony/twig-bundle (v4.4.25)
  - Downloading twig/extra-bundle (v3.3.1)
  - Downloading symfony/string (v5.3.0)
  - Downloading twig/string-extra (v3.3.1)
  - Removing zendframework/zend-escaper (2.6.1)
  - Removing voku/portable-ascii (1.5.6)
  - Removing ralouphie/getallheaders (3.0.3)
  - Removing phpoffice/common (0.2.9)
  - Removing pclzip/pclzip (2.8.2)
  - Removing nesbot/carbon (2.46.0)
  - Removing laravolt/avatar (3.2.1)
  - Removing kimai/user-bundle (1.3)
  - Removing intervention/image (2.5.1)
  - Removing illuminate/support (v7.30.4)
  - Removing illuminate/contracts (v7.30.4)
  - Removing illuminate/cache (v7.30.4)
  - Removing hslavich/oneloginsaml-bundle (v1.5.2)
  - Removing guzzlehttp/psr7 (1.8.1)
  - Removing doctrine/reflection (1.2.2)
  - Upgrading composer/package-versions-deprecated (1.11.99.1 => 1.11.99.2): Extracting archive
  - Upgrading symfony/flex (v1.12.2 => v1.13.3): Extracting archive
  - Upgrading symfony/service-contracts (v2.2.0 => v2.4.0): Extracting archive
  - Upgrading symfony/polyfill-php80 (v1.22.1 => v1.23.0): Extracting archive
  - Upgrading symfony/polyfill-php73 (v1.22.1 => v1.23.0): Extracting archive
  - Upgrading symfony/console (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading doctrine/cache (1.10.2 => 1.11.3): Extracting archive
  - Upgrading doctrine/annotations (1.12.1 => 1.13.1): Extracting archive
  - Upgrading doctrine/persistence (1.3.8 => 2.2.1): Extracting archive
  - Upgrading doctrine/inflector (1.4.4 => 2.0.3): Extracting archive
  - Upgrading doctrine/dbal (2.13.0 => 2.13.1): Extracting archive
  - Upgrading doctrine/common (2.13.3 => 3.1.2): Extracting archive
  - Upgrading doctrine/orm (2.7.5 => 2.9.2): Extracting archive
  - Upgrading symfony/routing (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/polyfill-ctype (v1.22.1 => v1.23.0): Extracting archive
  - Upgrading symfony/polyfill-intl-normalizer (v1.22.1 => v1.23.0): Extracting archive
  - Upgrading symfony/polyfill-intl-idn (v1.22.1 => v1.23.0): Extracting archive
  - Upgrading symfony/mime (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/http-foundation (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/http-client-contracts (v2.3.1 => v2.4.0): Extracting archive
  - Upgrading symfony/event-dispatcher (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/var-dumper (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading psr/log (1.1.3 => 1.1.4): Extracting archive
  - Upgrading symfony/debug (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/error-handler (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/http-kernel (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/finder (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/filesystem (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/dependency-injection (v4.4.21 => v4.4.25): Extracting archive
  - Installing symfony/polyfill-php81 (v1.23.0): Extracting archive
  - Upgrading symfony/config (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/var-exporter (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/cache-contracts (v2.2.0 => v2.4.0): Extracting archive
  - Upgrading symfony/cache (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/framework-bundle (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/stopwatch (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading laminas/laminas-zendframework-bridge (1.1.1 => 1.2.0): Extracting archive
  - Upgrading laminas/laminas-eventmanager (3.2.1 => 3.3.1): Extracting archive
  - Upgrading friendsofphp/proxy-manager-lts (v1.0.3 => v1.0.5): Extracting archive
  - Upgrading doctrine/migrations (2.3.3 => 3.1.2): Extracting archive
  - Upgrading symfony/doctrine-bridge (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/deprecation-contracts (v2.2.0 => v2.4.0): Extracting archive
  - Upgrading doctrine/doctrine-bundle (2.3.1 => 2.4.2): Extracting archive
  - Upgrading doctrine/doctrine-migrations-bundle (2.2.2 => 3.1.1): Extracting archive
  - Upgrading symfony/security-core (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading gedmo/doctrine-extensions (v2.4.42 => v3.0.5): Extracting archive
  - Upgrading phpstan/phpdoc-parser (0.4.14 => 0.5.4): Extracting archive
  - Upgrading jms/serializer (3.12.2 => 3.12.3): Extracting archive
  - Upgrading twig/twig (v3.3.0 => v3.3.2): Extracting archive
  - Upgrading symfony/translation-contracts (v2.3.0 => v2.4.0): Extracting archive
  - Upgrading symfony/twig-bridge (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/translation (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/options-resolver (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/asset (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading kevinpapst/adminlte-bundle (3.4 => 3.5): Extracting archive
  - Upgrading league/csv (9.6.2 => 9.7.1): Extracting archive
  - Upgrading league/html-to-markdown (4.10.0 => 5.0.0): Extracting archive
  - Upgrading mpdf/mpdf (v8.0.10 => v8.0.11): Extracting archive
  - Upgrading myclabs/php-enum (1.7.7 => 1.8.0): Extracting archive
  - Upgrading symfony/inflector (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/property-info (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading nelmio/cors-bundle (1.5.6 => 2.1.1): Extracting archive
  - Upgrading markbaker/matrix (2.1.2 => 2.1.3): Extracting archive
  - Upgrading markbaker/complex (2.0.0 => 2.0.3): Extracting archive
  - Upgrading phpoffice/phpspreadsheet (1.17.1 => 1.18.0): Extracting archive
  - Installing laminas/laminas-escaper (2.7.0): Extracting archive
  - Upgrading phpoffice/phpword (0.17.0 => 0.18.2): Extracting archive
  - Upgrading sensio/framework-extra-bundle (v5.6.1 => v6.1.5): Extracting archive
  - Upgrading symfony/mailer (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/amazon-mailer (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/css-selector (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/dotenv (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/expression-language (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/property-access (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/polyfill-intl-icu (v1.22.1 => v1.23.0): Extracting archive
  - Upgrading symfony/intl (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/form (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/google-mailer (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/http-client (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/mailchimp-mailer (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/mailgun-mailer (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading monolog/monolog (1.26.0 => 1.26.1): Extracting archive
  - Upgrading symfony/monolog-bridge (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/polyfill-intl-grapheme (v1.22.1 => v1.23.0): Extracting archive
  - Upgrading symfony/postmark-mailer (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/security-http (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/security-guard (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/security-csrf (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/security-bundle (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/sendgrid-mailer (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/serializer (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading symfony/polyfill-uuid (v1.22.1 => v1.23.0): Extracting archive
  - Upgrading symfony/uid (v5.2.6 => v5.3.0): Extracting archive
  - Upgrading symfony/validator (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/webpack-encore-bundle (v1.11.1 => v1.11.2): Extracting archive
  - Upgrading symfony/yaml (v4.4.21 => v4.4.25): Extracting archive
  - Upgrading symfony/twig-bundle (v4.4.20 => v4.4.25): Extracting archive
  - Upgrading twig/extra-bundle (v3.3.0 => v3.3.1): Extracting archive
  - Upgrading symfony/string (v5.2.6 => v5.3.0): Extracting archive
  - Upgrading twig/string-extra (v3.3.0 => v3.3.1): Extracting archive
Package symfony/inflector is abandoned, you should avoid using it. Use use `EnglishInflector` from the String component instead instead.
Package phpunit/php-token-stream is abandoned, you should avoid using it. No replacement was suggested.
Generating optimized autoload files
composer/package-versions-deprecated: Generating version class...
composer/package-versions-deprecated: ...done generating version class
102 packages you are using are looking for funding.
Use the `composer fund` command to find out more!

Run composer recipes at any time to see the status of your Symfony recipes.

Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 255
!!  PHP Warning:  include_once(/var/www/kimai2/vendor/gedmo/doctrine-extensions/lib/Gedmo/AbstractTrackingListener.php): failed to open stream: No such file or directory in /var/www/kimai2/var/cache/prod/ContainerP1L3yDl/getTimestampableListenerService.php on line 9
!!  PHP Warning:  include_once(): Failed opening '/var/www/kimai2/vendor/gedmo/doctrine-extensions/lib/Gedmo/AbstractTrackingListener.php' for inclusion (include_path='.:/usr/share/php') in /var/www/kimai2/var/cache/prod/ContainerP1L3yDl/getTimestampableListenerService.php on line 9
!!  PHP Warning:  include_once(/var/www/kimai2/vendor/gedmo/doctrine-extensions/lib/Gedmo/Timestampable/TimestampableListener.php): failed to open stream: No such file or directory in /var/www/kimai2/var/cache/prod/ContainerP1L3yDl/getTimestampableListenerService.php on line 10
!!  PHP Warning:  include_once(): Failed opening '/var/www/kimai2/vendor/gedmo/doctrine-extensions/lib/Gedmo/Timestampable/TimestampableListener.php' for inclusion (include_path='.:/usr/share/php') in /var/www/kimai2/var/cache/prod/ContainerP1L3yDl/getTimestampableListenerService.php on line 10
!!  PHP Warning:  include_once(/var/www/kimai2/vendor/kimai/user-bundle/Doctrine/UserListener.php): failed to open stream: No such file or directory in /var/www/kimai2/var/cache/prod/ContainerP1L3yDl/getFosUser_UserListenerService.php on line 9
!!  PHP Warning:  include_once(): Failed opening '/var/www/kimai2/vendor/kimai/user-bundle/Doctrine/UserListener.php' for inclusion (include_path='.:/usr/share/php') in /var/www/kimai2/var/cache/prod/ContainerP1L3yDl/getFosUser_UserListenerService.php on line 9
!!
!!
!!   [WARNING] Some commands could not be registered:
!!
!!
!!  In srcApp_KernelProdContainer.php line 1996:
!!
!!    Class 'FOS\UserBundle\Validator\Initializer' not found
!!
!!
!!  In MetadataSubscriber.php line 81:
!!
!!    Argument 1 passed to KimaiPlugin\AuditTrailBundle\Doctrine\MetadataSubscrib
!!    er::loadMetadataForObjectClass() must be an instance of Doctrine\Common\Per
!!    sistence\ObjectManager, instance of Doctrine\ORM\EntityManager given, calle
!!    d in /var/www/kimai2/vendor/gedmo/doctrine-extensions/src/Loggable/Loggable
!!    Listener.php on line 112
!!
!!
!!  In MetadataSubscriber.php line 81:
!!
!!    Argument 1 passed to KimaiPlugin\AuditTrailBundle\Doctrine\MetadataSubscrib
!!    er::loadMetadataForObjectClass() must be an instance of Doctrine\Common\Per
!!    sistence\ObjectManager, instance of Doctrine\ORM\EntityManager given, calle
!!    d in /var/www/kimai2/vendor/gedmo/doctrine-extensions/src/Loggable/Loggable
!!    Listener.php on line 112
!!
!!
!!  In MetadataSubscriber.php line 81:
!!
!!    Argument 1 passed to KimaiPlugin\AuditTrailBundle\Doctrine\MetadataSubscrib
!!    er::loadMetadataForObjectClass() must be an instance of Doctrine\Common\Per
!!    sistence\ObjectManager, instance of Doctrine\ORM\EntityManager given, calle
!!    d in /var/www/kimai2/vendor/gedmo/doctrine-extensions/src/Loggable/Loggable
!!    Listener.php on line 112
!!
!!
!!  In FileLoader.php line 38:
!!
!!    The mapping file "/var/www/kimai2/vendor/kimai/user-bundle/Resources/config
!!    /validation.xml" does not exist.
!!
!!
!!  In FileLoader.php line 38:
!!
!!    The mapping file "/var/www/kimai2/vendor/kimai/user-bundle/Resources/config
!!    /validation.xml" does not exist.
!!
!!
!!
!!
!!   [WARNING] Some commands could not be registered:
!!
!!
!!  In srcApp_KernelProdContainer.php line 1996:
!!
!!    Class 'FOS\UserBundle\Validator\Initializer' not found
!!
!!
!!  In MetadataSubscriber.php line 81:
!!
!!    Argument 1 passed to KimaiPlugin\AuditTrailBundle\Doctrine\MetadataSubscrib
!!    er::loadMetadataForObjectClass() must be an instance of Doctrine\Common\Per
!!    sistence\ObjectManager, instance of Doctrine\ORM\EntityManager given, calle
!!    d in /var/www/kimai2/vendor/gedmo/doctrine-extensions/src/Loggable/Loggable
!!    Listener.php on line 112
!!
!!
!!  In MetadataSubscriber.php line 81:
!!
!!    Argument 1 passed to KimaiPlugin\AuditTrailBundle\Doctrine\MetadataSubscrib
!!    er::loadMetadataForObjectClass() must be an instance of Doctrine\Common\Per
!!    sistence\ObjectManager, instance of Doctrine\ORM\EntityManager given, calle
!!    d in /var/www/kimai2/vendor/gedmo/doctrine-extensions/src/Loggable/Loggable
!!    Listener.php on line 112
!!
!!
!!  In MetadataSubscriber.php line 81:
!!
!!    Argument 1 passed to KimaiPlugin\AuditTrailBundle\Doctrine\MetadataSubscrib
!!    er::loadMetadataForObjectClass() must be an instance of Doctrine\Common\Per
!!    sistence\ObjectManager, instance of Doctrine\ORM\EntityManager given, calle
!!    d in /var/www/kimai2/vendor/gedmo/doctrine-extensions/src/Loggable/Loggable
!!    Listener.php on line 112
!!
!!
!!  In FileLoader.php line 38:
!!
!!    The mapping file "/var/www/kimai2/vendor/kimai/user-bundle/Resources/config
!!    /validation.xml" does not exist.
!!
!!
!!  In FileLoader.php line 38:
!!
!!    The mapping file "/var/www/kimai2/vendor/kimai/user-bundle/Resources/config
!!    /validation.xml" does not exist.
!!
!!
!!
!!   // Clearing the cache for the prod environment with debug
!!   // false
!!
!!  13:39:58 CRITICAL  [php] Uncaught Error: Argument 1 passed to KimaiPlugin\AuditTrailBundle\Doctrine\MetadataSubscriber::loadMetadataForObjectClass() must be an instance of Doctrine\Common\Persistence\ObjectManager, instance of Doctrine\ORM\EntityManager given, called in /var/www/kimai2/vendor/gedmo/doctrine-extensions/src/Loggable/LoggableListener.php on line 112 ["exception" => TypeError { …}]
!!
!!  In MetadataSubscriber.php line 81:
!!
!!    Argument 1 passed to KimaiPlugin\AuditTrailBundle\Doctrine\MetadataSubscrib
!!    er::loadMetadataForObjectClass() must be an instance of Doctrine\Common\Per
!!    sistence\ObjectManager, instance of Doctrine\ORM\EntityManager given, calle
!!    d in /var/www/kimai2/vendor/gedmo/doctrine-extensions/src/Loggable/Loggable
!!    Listener.php on line 112
!!
!!
!!  cache:clear [--no-warmup] [--no-optional-warmers] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command>
!!
!!
Script @auto-scripts was called via post-install-cmd
root@xyz-kimai:/var/www/kimai2#
`

**Comment by kevinpapst** [comment, marked as answer]
delete all plugin, then do `rm -r var/cache/*`, then run the update, instal plugins ...

**Comment by techsupportkoeln** [comment]
thx for the fast help, i'm now on version 1.15 with this workaround :)
